build-seed7

```diff
@@ -0,0 +1,181 @@
+#!/bin/sh
+# SH FILE: build-seed7
+#
+# Purpose   : Build the Seed7 system.
+# Created   : Friday, March 28 2025.
+# Author    : Pierre Rouleau
+# Time-stamp: <2025-03-28 17:44:46 EDT, updated by Pierre Rouleau>
+# ----------------------------------------------------------------------------
+# Module Description
+# ------------------
+#
+# Build and test the Seed7 system, using a single command.
+# Stop at the first error.
+# By default cleans, builds everything and runs tests.
+# There are options to do a dry-run or skip the clean and the test.
+# The help provides some info.
+
+# ----------------------------------------------------------------------------
+# Dependencies
+# ------------
+#
+# - Can only be executed from the root directory of Seed7 repo.
+# - Unix-like OS: Posix shell, printf, ls, sed, column.
+#
+
+# ----------------------------------------------------------------------------
+# Code
+# ----
+#
+#
+pgm_name="$(basename "$0")"
+makefile_name=""
+no_clean=""
+no_test=""
+dry_run=""
+
+print_usage()
+{
+    printf -- "\
+%s: List information of specified running process.
+
+ Usage: %s h|--help
+ • Print this help information.
+
+ Usage: %s [-n|--no-test] [makefile-name]
+ • Build Seed7.
+
+   Options:
+        -c | --no-clean: build only.  Do not run tests, which are
+                        executed by default.
+        -t | --no-test: build only.  Do not run tests, which are
+                        executed by default.
```

Comment:
Hi Pierre (@pierre-rouleau),

The descriptions for both of these options are the same. Might this have been an oversight, perhaps?

Kind Regards,
Liam

Reply:
Hi Liam (@celtic-coder ) you are correct I will fix it.  Thanks!